.github/CONTRIBUTING.md

```diff
@@ -67,6 +67,10 @@ Here are some common issues and resolutions:
   - Ensure any new markdown files only contain alphanumerics and dashes in their file names and have the same name as their parent directory.
   - If that looks okay, running `yarn test` locally should let you know what the issue is.
 
+## Some checks are not running how do I fix this?
+
+This is just a glitch. Take away 1 character and add it back to force the checks to rerun and this will fix the problem. Don't forget to give the checks a few minutes to run before doing this.
```

Comment:
Hmm, I do appreciate this though I don't think this is the right approach. Although this is may be what we end up doing, there's more going on under the hood for the issues we experience with a check failing.

Good effort but we need to revisit this. Not sure if that was a good explanation @sonnynomnom, but thinking we should close this PR and revisit

Reply:
Okay, so when the license/cla test isn't passing, it's referring to the original license we signed and the status for any of the repos regarding this can be checked up here: https://cla-assistant.io/

Usually when that test is failing, there's some issue with the email you're using to push up to Git. That's why I suggested using the global con gif command to set the email in the link you mentioned.

Reply:
@yangc95 It looks like some other checks don't run sometimes. Like on this PR.

![image](https://user-images.githubusercontent.com/92686703/159976531-33ceafc9-4ebb-4cac-8f6d-b07b87df0ed5.png)